components/bl-drawer/src/helpers.js

```diff
@@ -51,10 +53,8 @@ export function useDrawerActions(component, eventHandlers, drawerContentRef, dra
   }, [animationDuration, drawerContainerRef, drawerContentRef, onClose, placement, visibility]);
 
   useEffect(() => {
-    if (defaultOpen) {
-      setVisibility(true);
-    }
-  }, [])
+    setVisibility(defaultOpen);
+  }, [defaultOpen]);
```

Comment:
For now it looks not like `defaultOpen` but like one more variation of visibility. What if defaultOpen will be changed dynamically from handler?